app/Http/Controllers/QueryController.php

```diff
@@ -110,6 +116,9 @@ public function membershipTotals(Request $request)
 
                 if ($is_primary_student) {
                     $count_primary++;
+                    $member->update([
```

Comment:
I do not think it is necessary to make this nice as all this will get removed in the next PR anyways. This is only to compare the old and the new system